### PR TITLE
Updates for v2: Simplified configuration

### PR DESCRIPTION
**Pull request by justin808**
This change is [Reviewable](https://reviewable.io/reviews/shakacode/webpacker_lite/9)

### CODE REVIEW

**Comment by justin808** [comment]
@kaizencodes @robwise  @Conturbo  please review.

File: lib/webpacker_lite/env.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ def hot_loading?
     end
 
     def file_path
-      Rails.root.join("config", "webpack", "paths.yml")
+      Rails.root.join("config", "webpacker_lite.yml")
```
Comment (ledestin) [inline lib/webpacker_lite/env.rb:23]: Apparently, the same code is used in `configuration.rb`. Might be worth having it dried.

File: lib/webpacker_lite/helper.rb [hunk 2]
```diff
@@ -8,26 +8,24 @@ module WebpackerLite::Helper
...
   #   <%= javascript_pack_tag 'calendar', 'data-turbolinks-track': 'reload' %> # =>
-  #   <script src="/packs/calendar.js" data-turbolinks-track="reload"></script>
+  #   <script src="/public/webpack/development/calendar-1016838bab065ae1e314.js" data-turbolinks-track="reload"></script>
   #
   #   # In production mode:
   #   <%= javascript_pack_tag 'calendar', 'data-turbolinks-track': 'reload' %> # =>
-  #   <script src="/packs/calendar-1016838bab065ae1e314.js" data-turbolinks-track="reload"></script>
+  #   <script src="/public/webpack/production/calendar-1016838bab065ae1e314.js" data-turbolinks-track="reload"></script>
   def javascript_pack_tag(name, **options)
     javascript_include_tag(WebpackerLite::Manifest.lookup("#{name}#{compute_asset_extname(name, type: :javascript)}"), **options)
```
Comment (ledestin) [inline lib/webpacker_lite/helper.rb]: Hard to read, I'd consider extracting a method or two.

File: lib/webpacker_lite/helper.rb [hunk 3]
```diff
@@ -43,39 +41,22 @@ def javascript_pack_tag(name, **options)
...
-    manifested_names = []
-    default_case = args.any?
-    if default_case
-      args.flatten!
-      manifested_names += get_manifests(args)
-    end
-    asset_type = WebpackerLite::Env.hot_loading? ? :hot : :static
-    names = Array(kwargs[asset_type])
-    manifested_names += get_manifests(names)
+   return "" if WebpackerLite::Env.hot_loading? && !kwargs[enabled_when_hot_loading].presence
```
Comment (ledestin) [inline lib/webpacker_lite/helper.rb]: It looks like `enabled_when_hot_loading` might need to be a symbol.

File: test/helper_test.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    assert_equal @view.asset_pack_path("bootstrap.css"), "/webpack/bootstrap-c38deda30895059837cf.css"
+  end
+
+  def test_javascript_pack_tag
+    script = %(<script src="/webpack/bootstrap-300631c4f0e0f9c865bc.js"></script>)
+    assert_equal @view.javascript_pack_tag("bootstrap.js"), script
+  end
+
+  def test_stylesheet_pack_tag
+    style = %(<link rel="stylesheet" media="screen" href="/webpack/bootstrap-c38deda30895059837cf.css" />)
```
Comment (ledestin) [inline test/helper_test.rb]: I'd get rid of the variable `style` and just put the literal on 2nd line. If you want to keep it, I'd call it `expected_style` or something like that.

File: test/manifest_test.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+    error = assert_raises WebpackerLite::FileLoader::NotFoundError do
+      WebpackerLite::Manifest.lookup(asset_file)
+    end
+
+    assert_equal error.message, "Can't find #{asset_file} in #{manifest_path}. Is webpack still compiling?"
+  end
+
+  def test_lookup_success
+    asset_file = "bootstrap.js"
```
Comment (ledestin) [inline test/manifest_test.rb]: Usually, a variable in these tests is a literal that the result of the tested method is compared to. In this case, it's the literal of the tested method's argument. Breaks expectations.

File: lib/webpacker_lite/env.rb (hunk 1 above)
Comment (justin808) [reply lib/webpacker_lite/env.rb:23]: good call

File: lib/webpacker_lite/helper.rb (hunk 2 above)
Comment (justin808) [reply lib/webpacker_lite/helper.rb]: good

File: lib/webpacker_lite/helper.rb (hunk 3 above)
Comment (justin808) [reply lib/webpacker_lite/helper.rb]: correct

**Comment by justin808** [comment]
Hey! big question for @alexfedoseev  @robwise  @Judahmeek regarding hot reloading and webpacker_lite. 

Should webpacker_lite be putting in the value of the hot reloading server host:port as part of the rails helper, or depending on the configuration of that from the manifest.json and requiring the setting of the publicPath in both the webpack output and manifest plugin setup.

My gut tells me that it’s better to keep the webpack manifest simpler so it’s really the mapping of output names to possibly fingerprinted filenames.

File: lib/webpacker_lite/helper.rb (hunk 3 above)
Comment (ledestin) [reply lib/webpacker_lite/helper.rb]: So, the tests didn't catch it? Might be worth adding a test to cover that.

**Pull request merged**